js/components/form/daterange-picker.vue

```diff
@@ -131,11 +130,11 @@ export default {
         $(this.$els.endHidden).rules('add', {
             dateGreaterThan: this.$els.startHidden.id,
             required: (el) => {
-                return (this.startValue && !this.endValue) || (this.endValue && !this.startValue);
+                return (this.endValue && !this.startValue);
             },
```

Comment:
I think it somehow does not work as expected? Maybe it should be cast to Boolean?

Reply:
If I have an existing dataset with a complete temporal coverage and I delete the end date, I can't submit:
![image](https://github.com/user-attachments/assets/75b8107a-f317-4d4c-b67c-a0a46b64e78d)

Reply:
It took me a while to understand what I was checking wrong, but I finally managed to reproduce and fix it ;)

Thanks, it was indeed needing a proper bool value (`null` didn't cut it).